### DISCUSSION TITLE
greater_or_equal_than?

### DISCUSSION THREAD

**Discussion by greyblake** [greyblake/nutype#11] [Ideas]
I paste this [message](https://www.reddit.com/r/rust/comments/11128z3/comment/j8i1s2s/?context=3) from reddit here, I don't lose/forget about it:

> Nice!
> One thing that I would miss would be something like
> #[nutype(validate(greater_or_equal_than=0.0))]
> instead of min=0 as I find this notation more natural. So basically all the greater / greater_or_equal, lesser/ lesser_or_equal etc.
> For example, if you want a float that is strictly more that zero (because you might want to divide by it), then min=0 doesnt work as it accepts zero. I see that you can use with = but a named case for those usual cases would be more expressive imo. What do you think?

**Comment by raffimolero** [comment]
alternatives:
```
>= ge min
<= le max
< lt less lower under below
> gt more/greater higher over above
== eq equals is
!= ne not
```
personally i suggest adding a pair like less/greater or below/above. ideally the macro should reject those with both a `min` and a `greater` or a `max` and a `less`, or (an `equals` or `not`) and something else.

**Reply by greyblake** [reply]
Decided to use 
* `less`
* `less_or_equal`
* `greater`
* `greater_or_equal`

This validators are already available in `0.4.0-beta.2`. Please see https://docs.rs/nutype/0.4.0-beta.2/nutype/index.html#integer-validators
